m4/pbs_default_scheduler.m4

```diff
@@ -37,12 +37,7 @@
 
 AC_DEFUN([PBS_AC_DEFAULT_SCHEDULER],
 [
-  AC_MSG_CHECKING([for Default scheduler name])
```

Comment:
Well, IMO pbs_share.h should be generated from autoheader. autoheader provides the feature to solve such problem. 
May be moving pbs_share.h to autoheader is better approach.

In any case, will add the variable to pbs_share.h remove pbs_default_scheduler.m4